internal/cmd/install.go

```diff
@@ -185,11 +185,17 @@ func install(
 
 	// Install the tool
 	installPath := filepath.Join(installDir, tool.Name)
-	err = os.Rename(extractedToolPath, installPath)
+    err = utils.MoveFile(extractedToolPath, installPath)
 	if err != nil {
 		return
 	}
 
+    //⋅Set⋅file⋅permissions⋅to⋅be⋅executable
+    err = utils.SetPermissions(installPath)
+    if err != nil {
+        return
+    }
+
```

Comment:
The permissions are already set in https://github.com/toolctl/toolctl/blob/main/internal/cmd/shared.go#L145-L146. We don't need to do it twice, let's just remove it from shared.go.

Reply:
That line is executed earlier and the `fs.MoveFile` is creating a new file. 
I haven't passed the setting of the permissions into the `fs.MoveFile` to keep it reusable.